src/config.js

```diff
@@ -127,6 +127,14 @@ const parser = yargs
                 default: false,
                 group: 'Proxy options',
             },
+            'proxy-forward-user-attr': {
+                describe:
+                    'Forwarded attributes of the authenticated user to the upstream server.',
```

Comment:
Can we include example usage here? It's not obvious what this should look like. There might be an `example` attribute you can use

Reply:
Yep! Will do.